### PR TITLE
Cleanup tests for new LXC containers

### PR DESCRIPTION
**Pull request by naspeh**
- get rid `superdesk.tests.test_settings`, it outdated copy of default_settings with small changes, so I moved these settings to `superdesk.tests.update_config` (previously named `get_test_settings`)
- fixed `default_settings` a little (for example previous order for `CORE_APPS` broke the behave tests)
- update some test cases

### CODE REVIEW

**Comment by petrjasek** [comment]
this probably broke another test

**Comment by naspeh** [comment]
@petrjasek: yes, that test is written in a strange manner.

I updated description. I wanted to remove `superdesk.tests.test_settings` some time ago, I've decided to do it now. Seems all tests are passed!

So please review again.

File: tests/io/feeding_services/http_tests.py [hunk 1]
```diff
@@ -70,12 +69,19 @@ def test_get_expired_token(self):
         self.assertEquals('', TestFeedingService()._get_auth_token(provider))
 
     def test_fetch_token(self):
+        # TODO: need some rewriting
```
Comment (petrjasek) [inline tests/io/feeding_services/http_tests.py]: those credentials we were using don't work anymore, so you can probably get rid of it

Comment (naspeh) [inline tests/io/feeding_services/http_tests.py]: but the test name is `test_fetch_token`, so we need some replacement for that scenario, this why I'm keeping it there

**Pull request merged**